geometry_msgs/Pose center_pose
float32[] scale
int64 category
float32 confidence